File: Lista 01 - Java /Ex05/src/ex05/Ex05.java
package ex05;

import java.util.Scanner;

/**
 * @author bryan
 */
public class Ex05 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int quantidadeNotas = 3;
        double somaNotas = 0;
        String mensagem;

        for (int i = 0; i < quantidadeNotas; i++) {

            System.out.printf("Digite a %d ª nota: ", (i + 1));
            somaNotas += sc.nextDouble();
        }

        sc.close();

        double mediaAritmetica = somaNotas / quantidadeNotas;

        if (mediaAritmetica < 4) {
            mensagem = "Reprovado";
        } else if (mediaAritmetica < 6) {
            mensagem = "Exame especial";
        } else {
            mensagem = "Aprovado";
        }

        System.out.printf("A média do aluno(a) é %.2f e sua situação é %s \n", mediaAritmetica, mensagem);

    }

}

File: Prática/Lista 04 - C (Introdução)/Ex05/Ex05.c
#include "../Console.c"
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h> 

int main()
{
    int quantNotas = 3;
    float nota = 0;
    float somaNotas = 0;
    float media = 0;
    bool loop = false;

    for (int i = 0; i < quantNotas; i++)
    {
        clear();

        do{
            printf("Digite a %dº nota: ", i+1);
            scanf("%f",&nota);

            if(nota >= 0 && nota <=10){
                somaNotas += nota;
                loop = false;
            }else{
                clear();
                printf("Nota inválida, digite uma nota entre 0 e 10\n\n");
                loop = true;
            }

        } while(loop);

    }

    clear();

    media = somaNotas/quantNotas;

    printf("A média aritmética é %.2g, situação: ", media);

    if(media < 4.0) printf("Reprovado\n\n");
    else if(media < 6.0) printf("Exame Especial\n\n");
    else printf("Aprovado\n\n");

    return 0;
}

